decidim/decidim
# Add comments export to meetings

#### :tophat: What? Why?
The current implementation enables an administrator to export the comments on the meetings 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Meta decidim link:  https://meta.decidim.org/processes/roadmap/f/122/proposals/15791  

#### Testing
1. Go to admin panel
2. Navigate to a space that have meetings 
3. Click export and choose comments as csv , json or excel as per attached image 
4. As a normal  user navigate to profile / my data 
5. Request the data
6. in the attached archive you should see a file named : decidim-comments-comments-*** that will enclose the meeting comments as well

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)
![image](https://user-images.githubusercontent.com/105683/104155313-98c0e680-53ef-11eb-9b28-c340d3bd5096.png)


:hearts: Thank you!
